pilot/pkg/model/config.go

```diff
@@ -885,6 +903,22 @@ func (store *istioConfigStore) ServiceRoleBindings(namespace string) []Config {
 	return bindings
 }
 
+func (store *istioConfigStore) MeshRbacConfig() *Config {
+	meshRbacConfig, err := store.List(MeshRbacConfig.Type, "")
+	if err != nil {
+		log.Errorf("failed to get meshRbacConfig: %v", err)
+	}
+	if len(meshRbacConfig) > 1 {
+		log.Errorf("found %d meshRbacConfig, expecting only 1.", len(meshRbacConfig))
```

Comment:
As you going to ignore all policies with name is not 'default', this log can be 'warn' level, or even drop it completely.